portfolio/src/main/webapp/index.html

```diff
@@ -121,11 +121,13 @@
             </tr>
         </table>
     </div>
-    <form action="/data" method="POST">
+    <!-- POST request for comment below -->
+    <form action="/data" method="POST" onsubmit="return emptyBox()">
```

Comment:
Would be best if you explain in the comments why you are returning the value of `emptyBox()`, as its not a common thing to do in HTML attributes to have a return statement.

Reply:
done